build_scripts/appimage/AppRun

```diff
@@ -24,7 +29,19 @@ do
     fi
 done
 
+
 export PYTHONNOUSERSITE=1
-(PYTHONHOME="${APPDIR}/opt/python3.11" exec "${APPDIR}/opt/python3.11/bin/python3.11" "${APPDIR}/usr/src/monkey_island.py" $@)
+
+export run_island="${APPDIR}/opt/python3.11/bin/python3.11 ${APPDIR}/usr/src/monkey_island.py"
+
+if [ "$1" == "service_run" ]; then
+    (PYTHONHOME="${APPDIR}/opt/python3.11" ${run_island} "$@")
+else
+    # Save HOME and USER because capsh changes them. Until ubuntu adds --noenv option we have to rely on a workaround
+    export HOME_ORIGINAL=$HOME
+    export USER_ORIGINAL=$USER
+
+    (PYTHONHOME="${APPDIR}/opt/python3.11" sudo capsh --keep=1 --user=${USER_ORIGINAL} --inh=cap_net_bind_service --addamb=cap_net_bind_service -- -c "env HOME=${HOME_ORIGINAL} USER=${USER_ORIGINAL} ${run_island}" "$@")
```

Comment:
If someone wants to split this long command into separate variables feel free to try, I didn't manage to do it